TuringAdvisoryGroup/tryrolljs
# Make CollapsibleView cross-platform

## What's done

Made `CollapsibleView` ready for native. If we want a consistent API for the library, we have to get rid of class names & HTML elements (use `native-base` or `react-native` instead). It'll allow our library to accept the unified `style` prop for both `native` & `web`. Moreover, it'll make more components cross-platform out of the box.

## How to test

1. Run `yarn start`.
2. Go to `CollapsibleView` story. 
3. Make sure that it looks fine & works as expected.

## Tasks


## Demo (screenshots, recordings, etc.)
